packages/components/tag/src/Tag.styled.tsx

```diff
@@ -0,0 +1,70 @@
+import tokens from '@contentful/f36-tokens';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import type { EntityStatus, TagType } from './types';
+
+export interface TagProps {
+  tagType?: TagType;
+  style?: React.CSSProperties;
+  className?: string;
+  testId?: string;
+  children: React.ReactNode;
+  entityStatusType?: EntityStatus;
+}
+
+const tagTypeToStyles = ({ tagType }: { tagType: TagType }) => {
+  switch (tagType) {
+    case 'positive':
+      return css`
+        color: ${tokens.colorGreenBase};
+        background-color: ${tokens.colorGreenLightest};
+      `;
+    case 'primary':
+      return css`
+        color: ${tokens.colorBlueBase};
+        background-color: ${tokens.colorBlueLightest};
+      `;
+    case 'negative':
+      return css`
+        color: ${tokens.colorRedBase};
+        background-color: ${tokens.colorRedLightest};
+      `;
+    case 'warning':
+      return css`
+        color: ${tokens.colorOrangeDark};
+        background-color: #ffefd5; /* temporary hardcoded value until palette improvements */
+      `;
+    case 'secondary':
+      return css`
+        color: ${tokens.colorTextBase};
+        background-color: ${tokens.colorElementLight};
+      `;
+    case 'muted':
+      return css`
+        color: ${tokens.colorTextMid};
+        background-color: ${tokens.colorElementLightest};
+      `;
+    case 'primary-filled':
+      return css`
+        color: ${tokens.colorWhite};
+        background-color: ${tokens.colorPrimary};
+      `;
+    default:
+      return css``;
+  }
+};
+
+export const Container = styled.div<{ tagType: TagType }>`
```

Comment:
🤔 why styled components?

Reply:
it looked like a reasonable approach for me, happy to see a discussion about it

Reply:
I'm not aware of the differences between `css` and `styled` apis, but I think everyone is used to `css`, not a strong opinion, but worth considering consistency